mongomock/collection.py

```diff
@@ -254,6 +255,20 @@ def _copy_field(obj, container):
     return copy.copy(obj)
 
 
+def _validate_data_fields(data):
+    def recursive_none_key_check(data):
+        for key, value in data.items():
+            if "\0" in key:
```

Comment:
Please use single quotes for literal strings. `'\0'` (same in all your changes).

Reply:
Changed to using all single quotes